pyvista/core/pointset.py

```diff
@@ -1939,7 +1939,6 @@ def dimensions(self):
 
     @dimensions.setter
     def dimensions(self, dims):
-        """Set the dataset dimensions. Pass a length three tuple of integers."""
```

Comment:
I assume this was axed because this is just a setter anyway. But should we add a setting example to the getter docs then?

Reply:
Turns out there's a bug:

https://github.com/pyvista/pyvista/issues/4135

I'll add to the example there once I fix it.